nservicebus/serialization/newtonsoft.md

```diff
@@ -76,18 +78,30 @@ The `AddDeserializer` API can help transition between serializers. See the [Mult
 }}
 
 
-### Use a JsonConverter for XContainer and XDocument
+#### Use a JsonConverter for XContainer and XDocument
 
 
-#### The JsonConverter
+##### The JsonConverter
 
 This is a custom [JsonConverter](http://www.newtonsoft.com/json/help/html/CustomJsonConverter.htm) that replicates the approach used by the [core serializer](json.md).
 
 snippet: XContainerJsonConverter
 
 
-#### Use the JsonConverter
+##### Use the JsonConverter
 
 At configuration time the JsonConverter can then be used with the following.
 
 snippet: UseConverter
+
+### Use of $type requires an assembly qualified name
+
+The [core serializer](json.md) is registering a custom serialization binder to not require the assembly name to be present when inferring message type from the special [`$type` property supported by json.net](https://www.newtonsoft.com/json/help/html/SerializeTypeNameHandling.htm). Not having to specify the assembly name can be useful to reduce coupling when using the serializer for native integration scenarios like [demonstrated in this sample](/samples/sqltransport/native-integration).
```

Comment:
how much sense does it make to mention the core serializer when talking about v7? Can we just describe the current behavior instead?